central/node/datastore/dackbox/datastore/datastore_sac_test.go

```diff
@@ -262,9 +262,11 @@ func (s *nodeDatastoreSACSuite) TestGetNode() {
 				s.True(found)
 				s.NotNil(fetchedNode)
 
-				// Priority can have updated value, and we want to ignore it.
-				fetchedNode.Priority = s.testNodes[nodeID].Priority
-				s.Equal(*s.testNodes[nodeID], *fetchedNode)
+				if fetchedNode != nil {
```

Comment:
This prevents potential panic if `fetchedNode` is nil.

Reply:
For me to understand: `s.NotNil()` will not stop execution of the test when the subject is nil, correct?

Reply:
Correct, `NodNil` will do the assertion, print a message and then continue. The continuation in this case would lead to a panic.
It is not so bad as this is only a test code that would panic, but in this case it was an clear bug.